regression/goto-instrument/chain.sh

```diff
@@ -1,12 +1,14 @@
 #!/bin/bash
 
-SRC=../../../src
+src=../../../src
+goto_cc=$src/goto-cc/goto-cc
+goto_instrument=$src/goto-instrument/goto-instrument
 
-GC=$SRC/goto-cc/goto-cc
-GI=$SRC/goto-instrument/goto-instrument
+name=${@:$#}
+name=${name%.c}
 
-OPTS=$1
-NAME=${2%.c}
+args=${@:1:$#-1}
+
+$goto_cc -o $name.gb $name.c
+$goto_instrument $args $name.gb
 
-$GC $NAME.c -o $NAME.gb
-$GI $OPTS $NAME.gb
```

Comment:
It would be nice if the changes to chain.sh and existing test.desc could be in a separate commit (even though still part of the this pull request) to explain what those changes are about.